openage/codegen/coord.py

```diff
@@ -49,11 +49,11 @@ def format_members(formatstring, join_with=", "):
             """
             return join_with.join(formatstring.format(m) for m in member_list)
 
-        template_dict = dict(
-            members=member_list,
-            formatted_members=format_members,
-            camelcase="".join(member.title() for member in member_list),
-        )
+        template_dict = {
+            "members": member_list,
+            "formatted_members": format_members,
+            "camelcase": "".join(member.title() for member in member_list),
+        }
```

Comment:
what, why is the functional syntax no longer ok for pylint?

Reply:
Literals are much faster than the constructors in recent Python versions. 

https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/use-dict-literal.html